src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -708,9 +711,37 @@ public static void main(String[] args) {
 		t.stop().printSummary();
 	}
 
+	protected static Path getTempEDIFFile() {
+		String tempEDIFFileName = ".temp_edif_" + FileTools.getUniqueProcessAndHostID() + ".edf";
+		return FileSystems.getDefault().getPath(tempEDIFFileName);
+	}
+
+	public static boolean shouldParseInParallel(long size) {
+		return ParallelismTools.getParallel() && size > ParallelEDIFParser.MIN_BYTES_PER_THREAD;
```

Comment:
The way this reads, it might seem like `ParallelEDIFParser.MIN_BYTES_PER_THREAD * 2` might be more appropriate?  Not sure what the runtime difference is or how often files of size `MIN_BYTES_PER_THREAD+1 >= size >= MIN_BYTES_PER_THREAD*2` occur.

Reply:
Another idea is `ParallelEDIFParser.calcThreads() > 1`.

Reply:
Good shout!